docs/RELEASING.md

```diff
@@ -0,0 +1,77 @@
+# How to make a Release
+
+- [How to make a Release](#how-to-make-a-release)
+  - [Updating](#updating)
+    - [Updating Java Version](#updating-java-version)
+  - [Versioning](#versioning)
+  - [Changelog](#changelog)
+    - [Style](#style)
+    - [Template](#template)
+  - [Releasing Steps](#releasing-steps)
+
+
+## Updating
+
+New features and bugfixes PRs should always be made against `main` branch.
+
+### Updating Java Version
+
+When updating the java version don't forget to update also `jitpack.yml` to make use of the correct jdk because jitpack sometimes has a hard time defining which image it should use
```

Comment:
nit:

```suggestion
When updating the Java version don't forget to update also `jitpack.yml` to make use of the correct JDK because JitPack sometimes has a hard time defining which image it should use
```